<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarksDialog</class>
 <widget class="QDialog" name="BookmarksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookmarks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>252</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="EditTreeView" name="tree"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="addFolderButton">
       <property name="text">
        <string>Add Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addSeparatorButton">
       <property name="text">
        <string>Add Separator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditTreeView</class>
   <extends>QTreeView</extends>
   <header>utils/edittreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>search</tabstop>
  <tabstop>tree</tabstop>
  <tabstop>addFolderButton</tabstop>
  <tabstop>addSeparatorButton</tabstop>
  <tabstop>removeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>BookmarksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
